model-optimizer/extensions/front/CTCGreedyDecoderReplacement.py

```diff
@@ -89,3 +92,44 @@ def pattern(**kwargs):
 
     def replace_sub_graph(self, graph: Graph, match: dict):
         replace_ctc_greedy_decoder(graph, match)
+
+
```

Comment:
Please modify TF `CTCGreedyDecoder` extractor by adding `output_sparse_format = True` attribute. And apply these transformations if this attribute is set to True. You need to delete this attribute once the transformation is applied. Also, you need to delete this attribute in transformations above.
This approach prevents to apply these transformations consequently, i.e. CTCGreedyDecoderWithSpasreDense first and your new one after, that is incorrect.

Reply:
Updated, also added new test for this scenario.